tag.sh

```diff
@@ -126,7 +126,7 @@ git_tag() {
     local tag="$1"
     local commit="$2"
 
-    git tag -a "$tag" -s -m "Version $tag" "$commit"
+    git tag -a "$tag" -m "Version $tag" "$commit"
```

Comment:
I was having issues using `gpg` so disabled signing of the tags for now. If we think this is important, I'm happy to rollback this change